component.json

```diff
@@ -5,11 +5,47 @@
   "docsUrl":"https://github.com/elasticio/postgresql-component",
   "credentials": {
     "fields": {
-      "conString": {
-        "label": "Connection String",
+      "protocol": {
+        "viewClass": "TextFieldView",
+        "label": "Protocol",
+        "required": true,
+        "placeholder": "postgresql"
+      },
```

Comment:
The configuration field `protocol` doesn't have any sense, because the component supports the only sort of database: postgresql.
The connection string will always start with `postgresql://`
Example:
`postgres://username:password@localhost/database`

Reply:
yeah, this makes sense, especially after our conversation yesterday. I didn't realize that the jdbc: prefix wasn't a valid way of connecting to the database.